Source/WebInspectorUI/UserInterface/Models/PropertyPath.js

```diff
@@ -162,6 +162,12 @@ WI.PropertyPath = class PropertyPath
         return new WI.PropertyPath(object, component, this);
     }
 
+    appendPrivatePropertyName(object, propertyName)
+    {
+        var component = WI.PropertyPath.SpecialPathComponent.PrivatePropertyName + "[" + propertyName + "]";
```

Comment:
Can this be something like `appendPropertyName`'s `_canPropertyNameBeDotAccess` to output `.#privateName`?

Reply:
unfortunately no, as that results in `["#foo"]`

the goal here is to ensure that `isPathComponentImpossible` which prevents showing "Copy Path to Property" as something like `$0.#foo` is would not be valid since it's in a different scope

Reply:
Maybe `let` instead of `var` =)